infinispan-materialization/dv-mat.yaml

```diff
@@ -0,0 +1,93 @@
+apiVersion: teiid.io/v1alpha1
+kind: VirtualDatabase
+metadata:
+  name: mat-example
+spec:
+  replicas: 1
+  env:
+  - name: SPRING_TEIID_DATA_INFINISPAN_ISPN_USERNAME
+    valueFrom:
+          secretKeyRef:
+            name: infinispan-teiid
+            key: username
+  - name: SPRING_TEIID_DATA_INFINISPAN_ISPN_PASSWORD
+    valueFrom:
+          secretKeyRef:
+            name: infinispan-teiid
+            key: password
+  - name: SPRING_TEIID_DATA_INFINISPAN_ISPN_URL
+    valueFrom:
+          secretKeyRef:
+            name: infinispan-teiid
+            key: url
+  - name: SPRING_TEIID_DATA_INFINISPAN_ISPN_TRANSACTIONMODE
```

Comment:
are any of these default we can skip defining them?

Reply:
None have a default.  The secret values will depend upon the infinispan setup.  Defaulting to non durable XA or NON_XA could be a breaking change for existing usage of the infinispan translator.  The closest you could argue for is to have a default cache name as "default" - which is not really an expectation of infinspan anymore (https://issues.redhat.com/browse/TEIID-5882) but we could just document / assume that.